### PR TITLE
CodeReview of 12 from 19

### PR DESCRIPTION
**Pull request by NicoloCervo**
InciCsvCorrector.java and 
PhotoTester at lines 379 to 403

I think I've messed up something
just view the changes from the second to last commit

### CODE REVIEW

File: InciCsvCorrector.java [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+
+    /**
+     * eliminates line endings that break the ingredient entries writing the results in the corrected file
+     * @param source source file
+     * @param corrected result file
+     */
+    private static void correctInterruptedLines(File source,File corrected) throws IOException {
+
+        BufferedReader br = new BufferedReader(new FileReader(source));
+        BufferedWriter bw = new BufferedWriter(new FileWriter(corrected));
```
Comment (AndreaTonUnipd) [inline InciCsvCorrector.java:87]: Too short names for objects/variables can lead to a less readable code (mostly in more complex functions), you should be more explicit.

**Review by AndreaTonUnipd — COMMENTED** [review]
I found your class very readable overall, with a good methods modularity , and the right amount of comments in the most "complex to read" parts.

**Comment by NicoloCervo** [comment]
fixed

**Comment by AndreaTonUnipd** [comment]
Checklist:

- Hardcoding
-Appropriate comments/explanations
-Code readability 
-Modularity 
-Appropriate documentation
-Good Methods read order
-Indentation

**Pull request closed without merging**